src/Components/App/App.js

```diff
@@ -2,24 +2,37 @@ import movieData from '../../sampleData'
 import Movies from '../Movies/Movies'
 import './App.css'
 import { Component } from 'react'
+import selectedMovieData from '../../sampleData2'
+import MovieDetails from '../MovieDetails/MovieDetails'
 const logo = require('../../Images/cherry-tomato.png')
 
 class App extends Component {
   constructor() {
     super() 
     this.state = {
-      movies: movieData
+      movies: movieData,
+      singleMovie: ""
     } 
   }
 
+  seeMovieDetails = () => {
+    this.setState({movies: movieData, singleMovie: selectedMovieData})
```

Comment:
I am wondering if when we redefine the state here, should we reset movies back to an empty array. I know that we use this data for our map and potentially our find when we GET API data, so I am unsure about this.

Reply:
When this function is invoke, are any changes being made to the movieData array?  If not, does that need to be updated using setState?  If the only changes/updates are the singleMovie, you only need to update that piece of state.

Reply:
I recognize that some of this approach may be impacted as you implement Router :)